scripts/requirements-build.txt

```diff
@@ -2,4 +2,4 @@ ecdsa
 imagesize>=1.2.0
 intelhex
 pylint
-regex
+cddl-gen==0.3.0
```

Comment:
Do we need regex for some other reason, or only as a transitive dependency of cddl-gen? If we need it on its own, we should keep it here as a separate line, as well as in -extra.

Reply:
As far as I know, `regex` was only used by `cddl-gen`, also I couldn't find it used in any Python files in the codebase

Reply:
> As far as I know, regex was only used by cddl-gen, also I couldn't find it used in any Python files in the codebase

OK, great! That looks fine, then.

However, that's the sort of information that belongs in the commit message, not a github comment :).

Reply:
Sure! Done.